modules/db_instance/variables.tf

```diff
@@ -116,3 +116,8 @@ variable "tags" {
   description = "A mapping of tags to assign to all resources"
   default     = {}
 }
+
+variable "storage_encrypted" {
+  description = "Defines if the storage should be encrypted or not"
```

Comment:
Keep description as in official documentation - `Specifies whether the DB instance is encrypted.`. See here - https://www.terraform.io/docs/providers/aws/r/db_instance.html

Reply:
And also move it after `storage_type`.